Resources/Locale/en-US/administration/commands/spawnplayer.ftl

```diff
@@ -0,0 +1,10 @@
+cmd-spawnplayer-desc = Spawns the selected player as their current character at the ADMIN's position.
```

Comment:
Ditto

Reply:
Good catch. I'll put it in a _DV directory.